README.md

```diff
@@ -57,6 +58,9 @@ Now you are ready to run 🚀
 ```
 caddy run
 ```
+#### Fanout
+
+[Fanout](https://fanout.io/) is a cloud platform. To use Fanout you must purchase a paid account with a contract for Fastly's services.
```

Comment:
I think it's important to say that we only support the cloud version today. It would be nice if we can also extend the gem to support self-hosted Fanout.